src/date/format-properties.js

```diff
@@ -19,11 +19,12 @@ define([
  *
  * TODO Support other calendar types.
  */
-return function( pattern, cldr ) {
+return function( pattern, cldr, timeZone ) {
 	var properties = {
 			numberFormatters: {},
 			pattern: pattern,
-			timeSeparator: numberSymbol( "timeSeparator", cldr )
+			timeSeparator: numberSymbol( "timeSeparator", cldr ),
+			timeZone: timeZone
```

Comment:
If we don't add `timeZone` from `option.timeZone` to `properties.timeZone` we may not be able to do the tests. Can you clarify why we need to skip?